Admin/AbstractAdmin.php

```diff
@@ -766,16 +766,28 @@ public function getFilterParameters()
     {
         $parameters = array();
 
+        // if filter persister was configured with `true`, use the default persister (session)
+        if ($this->filterPersister === true) {
+            $this->filterPersister = new SessionFilterPersister($this->request->getSession());
```

Comment:
Couldn't this be done with DI, BTW? If no filterPersister is set, inject this default service?